instagram/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import './App.scss';
+
+import {SearchBar, PostContainer} from './components'
+import dummyData from './dummy-data'
+
+import PropTypes from 'prop-types'
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      dummyData
+    }
+  }
```

Comment:
Nicely done. I like that you're keeping the data in App level. This will help later when you need to filter the data based on user input.